server/pom.xml

```diff
@@ -38,7 +38,7 @@
         <artifactId>gson</artifactId>
         <version>2.8.6</version>
     </dependency>
-
+    
```

Comment:
nit: remove extra line

Reply:
![image](https://user-images.githubusercontent.com/37122992/86816404-c5b3cb80-c051-11ea-85f9-447357beea53.png)
Everything is spaced so that there's a line between each dependency, so removing an extra line would be imbalanced?

Reply:
You can leave it here then, but since the only change in that file was a new line, it's not really in scope for this PR (looks like it was leftover from changes that got reverted in this PR earlier).